src/logging.rs

```diff
@@ -125,13 +125,27 @@ impl log::Log for Logger {
     }
 
     fn log(&self, record: &log::Record<'_>) {
-        if self.enabled(record.metadata()) && record.target().starts_with("alacritty") {
-            let msg = format!(
-                "[{}] [{}] {}\n",
-                time::now().strftime("%F %R").unwrap(),
-                record.level(),
-                record.args()
-            );
+        if self.enabled(record.metadata()) &&
+           record.target().starts_with("alacritty")
+        {
+            let t = time::now();
+            let now = t.strftime("%F %R").unwrap();
```

Comment:
To get around the borrowing issue here, it should be possible to use the `time::strftime` method.

Something like this:
```rust
let now = time::strftime("%F %R", &time::now()).unwrap();
```

It's not much, but it prevents the need for a temporary variable that might confuse people.